addons/mail/static/src/core/common/message.xml

```diff
@@ -21,14 +21,18 @@
                         </div>
                         <t t-elif="message.isPending" t-call="mail.Message.pendingStatus"/>
                         <t t-elif="!message.is_transient">
-                            <small t-if="isActive and props.showDates" class="o-mail-Message-date text-muted opacity-75 o-smaller mt-2">
-                                <t t-esc="message.dateSimple"/>
-                            </small>
-                            <div t-else="">
-                                <MessageSeenIndicator
-                                    t-if="props.message.isSelfAuthored and props.thread?.hasSeenFeature"
-                                    message="props.message"
-                                    thread="props.thread"/>
+                            <div class="d-flex flex-column">
+                                <div t-if="isActive and props.showDates">
+                                    <small class="o-mail-Message-date text-muted opacity-75 o-smaller mt-2">
+                                        <t t-esc="message.dateSimple"/>
+                                    </small>
+                                </div>
+                                <div>
+                                    <MessageSeenIndicator
+                                        t-if="props.message.isSelfAuthored and props.thread?.hasSeenFeature"
+                                        message="props.message"
+                                        thread="props.thread"/>
+                                </div>
```

Comment:
Exclusive timestamp and seen indicator is intended with narrow size in sidebar. By removing it, the elements change position on hover and the message height is increased, so it doesn't look good:

![Jul-02-2024 11-38-44](https://github.com/odoo/odoo/assets/6569390/48db836b-84b0-4e24-a325-46efeac08a95)

We cannot solve this problem with the current design. I wouldn't reactively just place it somewhere else because no other better placement fits the current design.

Here are our good options, if we want to improve clarity of seen indicator:
1. Move timestamp inside bubble, like WhatsApp (sidebar on squash becomes fully owned by indicator so it's good)
2. Redesign completely seen indicator as listing of member avatars that act as a sort of divider, like FB messenger.

Reply:
This is solved by https://github.com/odoo/odoo/pull/171767